service/routes/docs.js

```diff
@@ -299,7 +341,10 @@ function route(req, res, next) {
 		.then(locals => {
 			template(locals.pageName)
 				.then(templFn => res.send(templFn(locals)))
-				.catch(() => next())
+				.catch(err => {
+					console.log(err);
+					next();
```

Comment:
Just checking we want to ignore this error. Fine if we do, I'm not _that_ familiar with this code